packages/audio-block/src/AudioBlock.tsx

```diff
@@ -0,0 +1,205 @@
+import {
+    AlignCenterPlugin,
+    AlignJustifyPlugin,
+    AlignLeftPlugin,
+    AlignRightPlugin,
+    BoldPlugin,
+    InitPlugin,
+    ItalicPlugin,
+    LoadingCircle,
+    PluginComposer,
+    Position,
+    ResetFormattingPlugin,
+    RichTextEditor,
+    StrikethroughPlugin,
+    TextStylePlugin,
+    UnderlinePlugin,
+} from '@frontify/fondue';
+import { Plugin, PluginProps } from '@frontify/fondue/dist/components/RichTextEditor/Plugins/Plugin';
+import { joinClassNames, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import {
+    Asset,
+    AssetChooserObjectType,
+    FileExtension,
+    useAssetUpload,
+    useBlockAssets,
+    useBlockSettings,
+    useEditorState,
+    useFileInput,
+} from '@frontify/app-bridge';
+import 'tailwindcss/tailwind.css';
+import { BlockSettings, TextPosition } from './types';
+import { AUDIO_EXTENSIONS, AUDIO_ID } from './settings';
+import { UploadPlaceholder } from './components/UploadPlaceholder';
+import { ItemToolbar } from './components/ItemToolbar';
+import { BlockAttachments } from './components/BlockAttachments';
+import { useEffect, useState } from 'react';
+
+const DEFAULT_CONTENT_TITLE = '[{"type":"heading3","children":[{"text":""}]}]';
+const DEFAULT_CONTENT_DESCRIPTION = '[{"type":"paragraph","children":[{"text":""}]}]';
+
+const customTitlePlugins = new PluginComposer();
+customTitlePlugins
+    .setPlugin([new InitPlugin(), new TextStylePlugin() as Plugin<PluginProps>])
+    .setPlugin([new BoldPlugin(), new ItalicPlugin(), new UnderlinePlugin(), new StrikethroughPlugin()])
+    .setPlugin([new AlignLeftPlugin(), new AlignRightPlugin(), new AlignCenterPlugin(), new AlignJustifyPlugin()])
+    .setPlugin([new ResetFormattingPlugin()]);
+
+export const AudioBlock = ({ appBridge }: BlockProps) => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [droppedFiles, setDroppedFiles] = useState<FileList | null>(null);
+    const isEditing = useEditorState(appBridge);
+    const [blockSettings, setBlockSettings] = useBlockSettings<BlockSettings>(appBridge);
+    const { blockAssets, deleteAssetIdsFromKey, updateAssetIdsFromKey } = useBlockAssets(appBridge);
+    const [openFileDialog, { selectedFiles }] = useFileInput({ multiple: false, accept: 'audio/*' });
+    const { designTokens } = useGuidelineDesignTokens();
+    const { title, description } = blockSettings;
+    const audio = blockAssets?.[AUDIO_ID]?.[0];
+
+    const [uploadFile, { results: uploadResults, doneAll }] = useAssetUpload({
+        onUploadProgress: () => !isLoading && setIsLoading(true),
+    });
+
+    const audioBlockClassNames = joinClassNames([
+        'tw-flex tw-flex-col tw-gap-3',
+        blockSettings.positioning === TextPosition.Above && 'tw-flex-col-reverse',
+    ]);
+
+    const audioTagClassNames = joinClassNames([
+        'tw-w-full tw-mt-5',
+        isEditing && 'group-hover:tw-border group-hover:tw-border-box-selected-inverse group-hover:tw-rounded',
```

Comment:
```suggestion
        isEditing && 'group-hover:tw-border group-hover:tw-border-box-selected-inverse',
```

+ there should be 1px offset between audio player and the purple outline. Maybe it is worth using outline instead of borders.

Reply:
fixed within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/270d443a55f7aca51132ff498bb0c1cdedbc3b59)

It's using outlines now, but i also needed to adjust the positioning of the ItemToolbar, to match the outline